packages/docs/docs/docs/zos_lib.md

```diff
@@ -9,32 +9,82 @@ this library can also be used directly to operate ZeppelinOS projects when a
 programmatic interface is preferred or more flexibility and lower-level
 access is required.
 
-## Installation
+## Install Node and NPM
 
 Let's install [Node.js](http://nodejs.org/) and
 [npm](https://npmjs.com/), which are the dependencies to get started. On their
 respective websites you will find specific instructions for your machine.
 
-Truffle is also required, and we'll be using [ganache](https://truffleframework.com/docs/ganache/quickstart) for local deployment, so let's install them and initialize a directory for our project:
+## Setting up your project
+
+We'll need to create a directory for our project and access it:
 
 ```console
-npm install --global truffle ganache-cli
 mkdir my-project
 cd my-project
-truffle init
+
+```
+
+Use `npm` to create a `package.json` file:
+```console
+npm init
+```
+
+This command will ask you for details about the project. For this very basic
+guide you can just press enter to accept the default values of each field.
+
+Next, we need to install Truffle. To do so, run the following command:
+
+```console
+npm install truffle@4.1.15
+```
+
+> Note: We are specifically installing Truffle 4.1.15, because ZeppelinOS 2.1 only supports Truffle 5 partially. Full support is expected in ZeppelinOS 2.2, which will be released soon._
```

Comment:
```suggestion
> Note: We are specifically installing Truffle 4.1.15, because ZeppelinOS 2.1 only supports that version of Truffle for its programmatic interface. Support for Truffle 5 is to be released in ZeppelinOS 2.2.
```